### PR TITLE
Task #221592: [WEB] Create components for each route

### PR DESCRIPTION
**Pull request by snehalsabade**
https://tracker.tekdi.net/issues/221592

### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [] Code is formatted as per format decided
* [] Updated acceptance criteria in tracker
* [] Updated test cases in test-cases-tracker

## Summary by CodeRabbit

- **New Features**
  - Introduced components to manage and display various learner details, including address, basic details, document checklist, education details, enrollment details, and PCR details.
  - Introduced navigation and rendering logic for learner profile and journey views.

- **Bug Fixes**
  - Corrected navigation routes from "Prerak" to "Learner" to ensure accurate routing within the application.

- **Refactor**
  - Updated access routes and conditions from `"program_coordinator"` to `"facilitator"`.
  - Modified route paths in the footer component to reflect learner-related routes.

### CODE REVIEW

File: apps/front-end/src/pages/PCusers/Learner/LearnerEducationDetails.js [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+          label={
+            <GetEnumValue
+              fontSize="14px"
+              key={index}
+              t={t}
+              enumOptionValue={item}
+              {...{ enumType, enumApiData }}
+            />
+          }
+        />
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/pages/PCusers/Learner/LearnerEducationDetails.js:35]: resolved

File: apps/front-end/src/pages/PCusers/Learner/LearnerJourneyDetails.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                          mr="5"
+                          alignItems="center"
+                        ></HStack>
+                      </HStack>
+                      {auditLogs.map((logs, i) => {
+                        return (
+                          <React.Fragment key={logs}>
+                            <HStack alignItems={"center"}>
+                              <Text width={"50px"}>{logs?.date}</Text>;
+                              <FrontEndTypo.Timeline
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/pages/PCusers/Learner/LearnerJourneyDetails.js:157]: resolved

File: apps/front-end/src/pages/PCusers/Learner/LearnerJourneyDetails.js [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+  const [auditMonth, setauditMonth] = React.useState([]);
+  const [auditYear, setauditYear] = React.useState([]);
+
+  const navigate = useNavigate();
+
+  React.useEffect(async () => {
+    const data = await enumRegistryService.listOfEnum();
+    setEnumOptions(data?.data ? data?.data : {});
+    agDetails();
+  }, [id]);
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/pages/PCusers/Learner/LearnerJourneyDetails.js:32]: resolved

File: apps/front-end/src/pages/PCusers/Learner/LearnerAddAddress.js [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+      edit_req_for_context_id: userId,
+    };
+    const result = await benificiaryRegistoryService.getEditRequest(obj);
+    if (result?.data?.length > 0) {
+      const fieldData = JSON.parse(result?.data?.[0]?.fields);
+      setRequestData(fieldData);
+    }
+    const data = await benificiaryRegistoryService.getOne(userId);
+    setbenificiary(data?.result);
+  }, []);
```
[the remark this answers is not in the document]
Comment (snehalsabade) [reply apps/front-end/src/pages/PCusers/Learner/LearnerAddAddress.js:33]: resolved

**Review by swapnilphalke03 — APPROVED** [review]

**Pull request merged**